### ISSUE TITLE
can DTW be negative?

### ISSUE THREAD

**Issue by yongjunshin** [Maghoumi/pytorch-softdtw-cuda#10]
Can DTW return a negative difference between the two sequences? (dist_func was not given. If so, as far as I know, euclidean distance works)

![image](https://user-images.githubusercontent.com/6459295/121176188-e48c9480-c896-11eb-95d2-a2d7c544a1a8.png)

In this image, those two sequences are compared by DTW cuda (dist_func was not given.)

...self.sdtw = SoftDTW(use_cuda=True, gamma=0.1)

diffs = self.sdtw(y_pred, y)
"""y_pred shape = (1000, 100, 1), y shape = (1000, 100, 1)"""

if diffs[0] < 0:   # current diffs[0] = -2.0720e+00
    plt.figure(figsize=(10, 5))
    plt.plot(y_pred[0, :, [0]].cpu().detach().numpy(), label="y_pred")
    plt.plot(y[0, :, [0]].cpu().detach().numpy(), label="y")
    plt.legend()
    plt.show() => above image.

**Comment by Maghoumi** [comment]
Yes, this issue duplicates #9. Please see the response by the original authors.
